modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/GatlingCodegen.java

```diff
@@ -160,14 +160,14 @@ public GatlingCodegen() {
     }
 
     /**
-     * Escapes a reserved word as defined in the `reservedWords` array. Handle escaping
-     * those terms here.  This logic is only called if a variable matches the reseved words
+     * Gatling does not need the models to have escaped words as it builds models dynamically instead of through
+     * an instance of the object.
      *
      * @return the escaped term
      */
     @Override
     public String escapeReservedWord(String name) {
-        return "_" + name;  // add an underscore to the name
+        return name;  // add an underscore to the name
```

Comment:
Maybe instead of overriding here, remove from AbstractScalaCodegen and override in other concrete Scala codegen classes

Reply:
The models for gatling are more for knowledge than for use.  I may remove them all together and just keep the toStringBody as that is really all that is needed.  Since all the objects are constructed from fields that are essentially coming from a CSV and translated through the session expression.  I left them there as it made sense at the time.